### DISCUSSION TITLE
Participation at Google Summer of code 2021

### DISCUSSION THREAD

**Discussion by mgogoulos** [mediacms-io/mediacms#187] [General]
Very happy to share that MediaCMS is one of the accepted programs for Google Summer of Code 2021! 

https://github.com/shubhank-saxena is going to work on adding new functionality on MediaCMS, that includes a command-line client, a dashboard for users with more detailed info, S3 storage support and many more!

**Comment by Dan1ell** [comment]
That's awesome! Congrats!
